build/js/intlTelInput-jquery.js

```diff
@@ -762,12 +762,35 @@
                 // listen on the document because that's where key events are triggered if no input has focus
                 var query = "";
                 var queryTimer = null;
+                // Was (/^[a-zA-ZÀ-ÿа-яА-Я ]$/.test(e.key)
+                function okToUse(char) {
+                    if (/^[a-zA-Z ]$/.test(char)) {
+                        return true;
+                    }
+                    if (char.codePointAt(1)) {
```

Comment:
Why is this bit here?

Reply:
That effectively limits it to a single character, like the original regex ... a bit weird.

Reply:
I thought that was what it was doing, just checking 👍